Firestore/Example/Podfile

```diff
@@ -10,63 +10,32 @@ pod 'Firebase/Core', '5.0.0'
 
 use_frameworks!
 
-pod 'FirebaseAuth', :path => '../../'
-pod 'FirebaseCore', :path => '../../'
-pod 'FirebaseFirestore', :path => '../../'
-
 target 'Firestore_Example_iOS' do
   platform :ios, '8.0'
 
-  # Test targets are below to avoid problems with duplicate symbols when
-  # building as a static library (see comments below).
-end
+  pod 'FirebaseAuth', :path => '../../'
```

Comment:
Did you intend to re-nest these here or is this just a bad merge? How does this work around all the duplicate class warnings we were getting from linking the framework into the example app?

Reply:
I don't intend to re-nest them. They ONLY get linked into the example app. I added -force_load to the Example app build `OTHER_LDFLAGS` so that unused C++ functions get linked in. Then the test target doesn't need them to build and finds them at runtime from the app.